README.md

```diff
@@ -56,8 +56,22 @@ celcl9aa-c3l7-7504-ca19-0c985e34ll8d
 In the Alfred worfklow in the upper right corner click the `[x]` icon and add the values from above to the corresponding value field
 
 [![img](https://i.imgur.com/Pe6nwey.jpg)](https://i.imgur.com/Pe6nwey.jpg)
+<!-- 
+JS: Remove this because notion_user_id is actually required in the code and this won't work.
```

Comment:
notion_user_id is only used for the recently viewed pages call that is shown when you first open the notion workflow. The code doesn't break if the value is not present, rather it just doesn't display anything until an actual query is given to the workflow.  So this comment is still valid.